index.html

```diff
@@ -191,6 +191,52 @@ <h3>A Special Time</h3>
                 <p>We might never know what happened before inflation, at the very beginning, because inflation erases everything that came before it. All the chaos and randomness of the primordial moment are swept away, forever out of our view.</p>
                 <p>“If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”</p>
             </section>
+            <section class="comment-section text">
+                <h2>Leave a comment</h2>
+                <form action="#" method="post">
+                    <ul>
+                        <li>
+                            <label for="name">Name</label>
+                            <input type="text" name="username" maxlength="20" id="name" placeholder="Fullname" required>
+                        </li>
+                        <li>
+                            <label for="email">Email</label>
+                            <input type="text" name="email" maxlength="20" id="email" placeholder="Email address" required>
+                        </li>
+                        <li><label for="comment">Comment</label>
+                            <textarea name="comment" maxlength="250" id="comment" cols="30" rows="10" placeholder="Type your comment here…" required></textarea>
+                        </li>
+                        <li>
+                            <span>What is your overall opinion about this article?</span>
+                            <ul class="opinion-option">
+                                <li>
+                                    <label>
+                                        <input type="radio" name="opinion"> Positive
+                                    </label>
+                                </li>
+                                <li>
+                                    <label>
+                                        <input type="radio" name="opinion"> Negative
+                                    </label>
+                                </li>
+                                <li>
+                                    <label>
+                                        <input type="radio" name="opinion"> Neutral
+                                    </label>
+                                </li>
+                            </ul>
+                        </li>
+                        <li>
+                            <label>
+                                <input type="checkbox" name="subscribe"> <strong> I want to subscribe to the newsletter</strong>
+                            </label>
```

Comment:
## Changes Requested

-  add a `for=""` attribute to all `label` elements.
-  add `id` attribute to all input elements (so that you can target them with the `for` attribute of the `label` elements).